schema/material/primitive/combinations/merge.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
Let's isolate combinations and modifications into `material/operations` or something like that

Reply:
Maybe `materials/operations/primitive`